internal/backends/winit/lib.rs

```diff
@@ -235,8 +235,16 @@ impl BackendBuilder {
     /// slint::platform::set_platform(Box::new(backend));
     /// ```
     pub fn build(self) -> Result<Backend, PlatformError> {
-        let event_loop_builder =
-            self.event_loop_builder.unwrap_or_else(winit::event_loop::EventLoop::with_user_event);
+        let event_loop_builder = self.event_loop_builder.unwrap_or_else(|| {
+            #[allow(unused_mut)]
+            let mut default_builder = winit::event_loop::EventLoop::with_user_event();
+            #[cfg(all(feature = "muda", target_os = "macos"))]
+            winit::platform::macos::EventLoopBuilderExtMacOS::with_default_menu(
```

Comment:
So if the user set an `event_loop_builder` (using the private API) the menu won't be visible. I believe we probably should disable the default_menu even if there is already an event_loop_builder. No?